### PR TITLE
Custom filters and transfomers

### PR DESCRIPTION
**Pull request by asafshen**
examples for custom filters & transformers
- InRange (number filter)
- ReverseList (entire-list transformer - e.i. get the whole list as an argument rather then works on each argument)
- StripChars - (string transformer)strip chars from prefix/suffix of the string

### CODE REVIEW

**Comment by dorsha** [comment]
@meirwah ?

File: Scripts/script-InRange.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+enabled: true
+args:
+- name: left
+  required: true
+  description: value to check if in range (e.g. 4)
+- name: right
+  required: true
+  description: range to check against with the form of from,to (e.g. 2,8)
+scripttarget: 0
+runonce: false
```
Comment (meirwah) [inline Scripts/script-InRange.yml]: no need for this right?

File: Scripts/script-ReverseList.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  the argument as a list  (note the \"entirelist\" tag)\n "
+enabled: true
+args:
+- name: value
+  required: true
+  default: true
+  description: List to reverse
+  isArray: true
+scripttarget: 0
+runonce: false
```
Comment (meirwah) [inline Scripts/script-ReverseList.yml]: no need for this right?

File: Scripts/script-StripChar.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+enabled: true
+args:
+- name: value
+  required: true
+  description: text to trim (e.g. "~!!~www.mydomain.com~!~!~")
+- name: chars
+  required: true
+  description: charecters to stripped from the text (e.g. "!~")
+scripttarget: 0
+runonce: false
```
Comment (meirwah) [inline Scripts/script-StripChar.yml]: no need for this right?

File: Scripts/script-InRange.yml (hunk 1 above)
Comment (asafshen) [reply Scripts/script-InRange.yml]: it imported automatically with the script
did we remove this feature?

**Pull request merged**